README.md

```diff
@@ -10,6 +10,49 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+What does OpenCraft Instance Manager do and how to use it
+---------------------------------------------------------
+
+This section focuses on how to use the web interface, as opposed to how to install the code,
+debug it or develop it (see next sections for that).
+
+Ocim is a web interface to manage a list of Open edX servers. With some clicks you can create
```

Comment:
I'd say Ocim is more of a continuous delivery tool -- the web console is just a small part of it.

Reply:
Well, I wouldn't define it as CI either… it's not rather „continuous“ yet. I left the definition at the top of the README untouched, it's better; and here I'll just say that it includes an interface.